File: src/scala/Q2010_ElectricalOutlets.scala
object Q2010_ElectricalOutlets {
  def main(args: Array[String]) : Unit = {
    val n = readInt
    val availablePlug = for (i <- 1 to n) yield {
      readInt
    }
    println(availablePlug.sum - n + 1)
  }
}

File: src/java/Q2010_ElectricalOutlets.java
import java.util.Scanner;

public class Q2010_ElectricalOutlets {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int n = scan.nextInt();

		long availablePlug = 1;
		for (int i = 0; i < n; i++) {
			availablePlug--;
			availablePlug += scan.nextInt();
		}

		System.out.println(availablePlug);
		scan.close();
	}
}
